src/System.Net.Security/src/System/Net/Security/SslStreamInternal.cs

```diff
@@ -22,9 +23,7 @@ internal class SslStreamInternal
         private static readonly AsyncProtocolCallback s_readFrameCallback = new AsyncProtocolCallback(ReadFrameCallback);
 
         private const int PinnableReadBufferSize = 4096 * 4 + 32;         // We read in 16K chunks + headers.
```

Comment:
If you're removing PinnableBufferCache, then maybe these constants should be removed or renamed?

Reply:
I have made this change, ReadBufferSize.